### DISCUSSION TITLE
Ribbon UI in File Explorer covers top folders

### DISCUSSION THREAD

**Discussion by AX-Procyon** [valinet/ExplorerPatcher#399] [General]
When I check file explorer setting "Disable Windows 11 command bar", the Ribbon UI appears and functions correctly. However, it covers the top line of folders when extended. 

This is what it looks like when the ribbon is minimized:
![Screenshot 2021-11-17 102215](https://user-images.githubusercontent.com/31220564/142101761-fff8598a-cd98-48be-82f8-537382f9e44e.jpg)

However, when I extend the ribbon, you can see it covers Desktop and Documents folders:
![Screenshot 2021-11-17 102201](https://user-images.githubusercontent.com/31220564/142102283-e55d978c-c642-4e13-b0d4-20863ce48455.jpg)
This does not only happen in This PC folder. All folders have the same problem. 
In past Windows builds, we have a "pin ribbon" button on the right side, next to "expand ribbon" button. This would bump the whole UI down and make the ribbon not covering anything. But in Windows 11 the button is gone, or at least doesn't function. Can this be patched? Thanks in advance.

**Comment by huggy020** [comment]
I'm also having the ribbon covering the top folders. Anybody else?

**Reply by highlanderc** [reply]
same here... any solution to this?

**Comment by Poopooracoocoo** [comment]
Does double clicking the ribbon tab pin the ribbon?

**Comment by highlanderc** [comment]
I don't think its a ribbon problem. the problem is that the folder pane is moved / offset to the top of the window, instead of being under or relative to the toolbars.

**Comment by highlanderc** [comment]
Solution for the future:
https://winbuzzer.com/2021/12/21/how-to-get-the-classic-file-explorer-with-ribbon-back-in-windows-11-xcxwbt/

follow that.
